examples/server/webui/src/utils/storage.ts

```diff
@@ -116,6 +117,12 @@ const StorageUtils = {
     });
     return conv;
   },
+  async setExtraContext(extraContext: string): Promise<void> {
+    db.extraContext = extraContext
+  },
+  async getExtraContext(): Promise<string> {
```

Comment:
Either define a default value, or explicitly tell that it is undefine-able:

```suggestion
  async getExtraContext(): Promise<string | undefined> {
```

Reply:
As extraContext will be moved in app.context, this function will be removed.